text/concepts.tex

```diff
@@ -230,6 +230,8 @@ \section{Restricting memory access with compressed bounds}
 	\caption[label=tbl:primary_permissions]{CHERIoT primary permissions}
 \end{table}
 
+\amanda{The HTML looks weird here, and could use a couple of newlines between the end of the table and the next paragraph.The PDF also looks weird for a different reason-- you start describing the items before the table appears.}
```

Comment:
That's a CSS thing, easy to fix.  The PDF moves images and tables to the next page if they'd overlap the end.  As long as we keep to the style that the first reference to a floating element is before the float appears, that's fine.